### DISCUSSION TITLE
Unable to upgrade Backblaze cask

### DISCUSSION THREAD

**Discussion by jaredledvina** [Homebrew/discussions#1824] [Casks]
```
❯ brew upgrade backblaze
Error: Not upgrading 1 `installer manual` cask.
backblaze
```

Looks like that's setup in https://github.com/Homebrew/homebrew-cask/blob/508d25842f3279561e47a011c2548cecd73e1c8a/Casks/backblaze.rb#L18 

Is there a way to work around that? Or to get Homebrew to stop trying and failing to upgrade this specific cask? I usually use `brew upgrade --greedy` but that always fails now with this same error. 

Thanks!

**Comment by weakish** [comment]
`installer manual` casks can only be reinstalled:

```sh
brew reinstall --cask backblaze
open /usr/local/Caskroom/backblaze/VERSION/Backblaze\ Installer.app
```

see also #1514
